### DISCUSSION TITLE
Add a way to skip osu! startup intro

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#23933] [Ideas] [answered]
I didn't think osu! needed this but now I think having a way to skip the intro sequence at the start of the game would be pretty nice.

As an example sometimes I get disconnected from a room I'm hosting and then after I join back in I can no longer send messages. DMs don't send either. So I usually have to restart the game to fix that and when I have to do that I wanna do it as quickly as possible so I don't leave my room waiting in confusion. That is when I would skip the osu! intro sequence at the start of the game which usually takes like 5 seconds.
Sometimes people can be pretty impatient and I think osu! should accustom for those times and offer that option.

Maybe with <kbd>Esc</kbd> which skips a part of the intro sequence in a smooth way.

**Comment by peppy** [comment, marked as answer]
We can look to fix your bug, rather than fix the fact you need to restart the game.

The intro will not be skippable. If you want, you can change it to a different intro type from settings.

Please try reporting your bug with sending messages along with logs, we'd be interested to look into that.
